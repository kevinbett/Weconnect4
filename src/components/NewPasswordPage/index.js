import React from 'react';
import Navigation from '../Navigation/index';
import NewPasswordForm from '../NewPasswordForm';

const NewPasswordPage = () => (

   <div>
       <Navigation />
       < NewPasswordForm/>
      
   </div>
);

export default NewPasswordPage;