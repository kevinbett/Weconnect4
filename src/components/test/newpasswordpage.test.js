import NewPasswordPage from '../NewPasswordPage/index'
import { shallow } from "enzyme"

describe('New password page',() => { 
    it ('New password page renders', () => { 
        expect(shallow(<NewPasswordPage/>)).toMatchSnapshot()
    });
} )