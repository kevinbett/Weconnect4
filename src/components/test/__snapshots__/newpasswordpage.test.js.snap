// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`New password page New password page renders 1`] = `
<div>
  <Navigation />
  <NewPasswordForm />
</div>
`;

exports[`Regi business page Register business page renders 1`] = `
<div>
  <Navigation />
  <NewPasswordForm />
</div>
`;

exports[`Register business page Register business page renders 1`] = `
<div>
  <Navigation />
  <RegisterBusinessForm />
</div>
`;
